backend/CuratedLists/views.py

```diff
@@ -133,10 +133,13 @@ def all_curations(request):
         response = HttpResponseNotAllowed(['GET'])
     else:
         try:
-            curations = Curation.objects.all()
+            curations = json.loads(serializers.serialize('json', Curation.objects.all())
+            for cur in curations:
+                cur['fields']['profile'] = json.loads(serializers.serialize("json",
+                Author.objects.filter(id=cur['fields']['author'])))
```

Comment:
Indent with the parantheses:
```python
cur['fields']['profile'] = json.loads(serializers.serialize(
  "json", Author.objects.filter(id=cur['fields']['author'])
))
```

Reply:
Since there's only one element in the serializer, why not attach it as a JSON object instead of list?